index.html

```diff
@@ -182,7 +193,7 @@ <h2 class="team-name">Colin Heshmat</h2>
         </div>
 
         <div class="team-item col-xs-12 col-sm-6 col-lg-4">
-          <img data-name='annlin' src="assets/avatars/giphy.gif" alt="Annlin" class="team-avatar" height="175px">
+          <img data-name='annlin' src="assets/avatars/annlin.png" alt="Annlin" class="team-avatar" height="175px">
```

Comment:
Why'd you add `height=175px` here? It makes the avatar look squashed and takes it out of alignment with the other avatars.